### DISCUSSION TITLE
NetMount：挂盘小工具，挂载 AList 到本地 - 夏沫花火zzz🌙 (Muska_Ami)のLife

### DISCUSSION THREAD

**Discussion by giscus** [Muska-Ami/muska-ami-blog#80] [General]
# NetMount：挂盘小工具，挂载 AList 到本地 - 夏沫花火zzz🌙 (Muska_Ami)のLife

什么是 NetMount官网介绍：  统一管理和挂载云存储设施  其实是一个基于 AList 和 Rclone 的开源网络盘挂载工具（个人感觉管理功能偏少），支持挂载为本地盘，而且无需付费。前端基于 Tauri 开发。 官网：NetMount  使用 NetMount 挂载自家 AList我在自己家部署了 MinIO 和 AList，用来做 NAS，但是一直没找到好的挂载软件。NetMount 很

https://blog.1l1.icu/2024/07/14/netmount-gua-pan-xiao-gong-ju-gua-zai-alist-dao-ben-di/

**Comment by ArcticLampyrid** [comment]
感觉不错，打了个 AUR 包：https://aur.archlinux.org/packages/netmount

**Reply by Muska-Ami** [reply]
软件确实不错，不过作者貌似不是很花心思去回复Issue，之前翻过一下开启的Issue列表，大部分都没有回复
大概是作者比较忙吧
